src/features/common/WebappButton/index.tsx

```diff
@@ -38,7 +38,11 @@ function WebappButton({
           {otherProps.icon !== undefined && (
             <div className={styles.webappButtonIcon}>{otherProps.icon}</div>
           )}
-          <div className={styles.webappButtonLabel}>{otherProps.text}</div>
+          <div
+            className={`${styles.webappButtonLabel} ${otherProps.buttonClasses}`}
+          >
+            {otherProps.text}
+          </div>
```

Comment:
Is buttonClasses needed for the label? It was intended to be applied at the outer `button` element.

Reply:
I've removed the class, seems to be added by mistake